packages/react-juce/src/lib/BlueprintBackend.ts

```diff
@@ -123,6 +168,21 @@ export class ViewInstance {
     // of a css prop, first convert it to kebab-case
     propKey = cssPropsMap[propKey] || propKey;
 
+    // convert provided color string to alpha-hex code for JUCE
+    if(propKey.includes("color")){
```

Comment:
Yep, right idea exactly.

Instead of searching for `color` though I think we should maintain a list of known color properties. For example,

```
border-color
background-color
color
```

and do a quick `if (isColorProperty(propKey))` here. That way if the end user writes a custom native component with some custom property named `tshirtColor` or something we don't trip this check accidentally

Reply:
ah sweet makes sense will make this change shortly